app/models/manageiq/providers/autosde/inventory/parser/storage_manager.rb

```diff
@@ -43,6 +45,28 @@ def storage_resources
     end
   end
 
+  def san_addresses
+    collector.san_addresses.each do |san_addresses_array|
+      san_addresses_array.each do |san_address_hash|
```

Comment:
Do you need the array of arrays?  Or could you just `.flat_map` in the collector?

Reply:
@agrare, I modified the code to use flat_map.